src/fdrs_gateway.h

```diff
@@ -193,3 +215,16 @@ void loopFDRS()
   }
 }
 
+// "Skeleton Functions related to FDRS Actions"
```

Comment:
Skeleton Functions needed to allow these functions to be defined in ACTIONS even when the features are not enabled.   Example:  If LoRa is not enabled that the entire file that has the LoRa functions is excluded from compilation but actions still include broadcastLoRa so need that function present for compilation to succeed.